From cfe07affad708cf7e70decd86bb3cca64e9fca86 Mon Sep 17 00:00:00 2001
From: Ben Mason <benmason@google.com>
Date: Mon, 27 Jun 2022 14:21:01 +0000
Subject: [PATCH 42/59] Updating XTBs based on .GRDs from branch 5005

Change-Id: I17782c02f17621fa514cfb29abee2836213415b2
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3726410
Auto-Submit: Ben Mason <benmason@chromium.org>
Bot-Commit: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/5005@{#1201}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 chrome/app/resources/generated_resources_ne.xtb                 | 2 +-
 chrome/credential_provider/gaiacp/strings/gaia_resources_my.xtb | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/chrome/app/resources/generated_resources_ne.xtb b/chrome/app/resources/generated_resources_ne.xtb
index 3b97345b7229..7f5558886721 100644
--- a/chrome/app/resources/generated_resources_ne.xtb
+++ b/chrome/app/resources/generated_resources_ne.xtb
@@ -6698,7 +6698,7 @@
 <translation id="7614260613810441905">कुनै साइटले तपाईंको डिभाइसका फाइल वा फोल्डरहरू सम्पादन गर्न खोज्दा सोध्नुहोस् (सिफारिस गरिएको)</translation>
 <translation id="761530003705945209">Google ड्राइभमा ब्याकअप गर्नुहोस् जुनसुकै बेला सजिलैसँग आफ्नो डेटा पुनर्स्थापना गर्नुहोस् वा यन्त्र बदल्नुहोस्। तपाईंको ब्याकअपमा एपको डेटा समावेश हुन्छ। तपाईंका ब्याकअपहरू Google मा अपलोड गरी तपाईंको Google खाताको पासवर्ड प्रयोग गरेर इन्क्रिप्ट गरिन्छन्।</translation>
 <translation id="7615365294369022248">खाता थप्ने क्रममा कुनै त्रुटि भयो</translation>
-<translation id="7616214729753637086">यन्त्र दर्ता गर्दै...</translation>
+<translation id="7616214729753637086">डिभाइस दर्ता गरिँदै छ...</translation>
 <translation id="7617263010641145920">Play Store अन गर्नुहोस्</translation>
 <translation id="7617366389578322136">"<ph name="DEVICE_NAME" />" मा जडान गरिँदै</translation>
 <translation id="7617648809369507487">आवाज नआउने म्यासेजिङ प्रयोग गर्नुहोस्</translation>
diff --git a/chrome/credential_provider/gaiacp/strings/gaia_resources_my.xtb b/chrome/credential_provider/gaiacp/strings/gaia_resources_my.xtb
index 0cf012125d68..fc7eb723b99f 100644
--- a/chrome/credential_provider/gaiacp/strings/gaia_resources_my.xtb
+++ b/chrome/credential_provider/gaiacp/strings/gaia_resources_my.xtb
@@ -7,7 +7,7 @@
 <translation id="2515346402363002066">သတ်မှတ်ပေးထားသည့်အချိန် ကုန်သွားပါပြီ။ သင့်အလုပ်သုံးအကောင့်နှင့် လက်မှတ်ထိုးဝင်ပါ။</translation>
 <translation id="2549902055700841962">သင့်အလုပ်သုံး အကောင့်သို့ လက်မှတ်ထိုးဝင်၍ မရပါ။ သင့်စီမံခန့်ခွဲသူထံ ဆက်သွယ်ပါ။</translation>
 <translation id="2566603360883977759">ဤအီးမေးလ်ဖြင့် လက်မှတ်ထိုးဝင်ခွင့်မပြုပါ။ အလုပ် (သို့) ကျောင်းအတွက် သင်အသုံးပြုသောအကောင့်ကို ပြန်စမ်းကြည့်ပါ။ လက်မှတ်ထိုးဝင်၍မရသေးပါက သင်၏စီမံခန့်ခွဲသူထံ ဆက်သွယ်ပါ။</translation>
-<translation id="2844349213149998955">ဤစက်ပေါ်တွင် ကိုယ်ရေးကိုယ်တာအကောင့်နှင့် လက်မှတ်ထိုးဝင်ခြင်းကို ခွင့်မပြုပါ။ အလုပ်သုံး အကောင့်နှင့် လက်မှတ်ထိုးဝင်ပါ။</translation>
+<translation id="2844349213149998955">ဤစက်ပေါ်တွင် ကိုယ်ပိုင်အကောင့်ဖြင့် လက်မှတ်ထိုးဝင်ခြင်းကို ခွင့်မပြုပါ။ အလုပ်သုံး အကောင့်ဖြင့် ဝင်ပါ။</translation>
 <translation id="3217145568844727893">သင့်လက်ရှိ Windows စကားဝှက် မထည့်ဘဲ ရှေ့ဆက်ပါက ဤစက်ပေါ်ရှိဒေတာများ အပြီးဆုံးရှုံးသွားနိုင်သည်။</translation>
 <translation id="3306357053520292004">ဤကွန်ပျူတာပေါ်ရှိ အသုံးပြုသူတစ်ဦးကို ဤအကောင့်သုံး၍ ထည့်ထားပြီး ဖြစ်ပါသည်။ အခြားအကောင့်တစ်ခုဖြင့် လက်မှတ်ထိုးဝင်ရောက်ပါ။</translation>
 <translation id="3355053591933237049">သင့်စက်ကို အင်တာနက်ချိတ်ဆက်ထားခြင်း ရှိမရှိ စစ်ဆေးပြီး ထပ်စမ်းကြည့်ပါ</translation>
-- 
2.37.0

